builds/e2e/connectivity.yaml

```diff
@@ -122,9 +131,9 @@ jobs:
         displayName: 'Download Images Artifacts'
         inputs:
           buildType: specific
-          project: $(vsts.project)
-          pipeline: $(images.build)
-          branchName: $(images.branchName)
+          project: $(resources.pipeline.images.projectID)
+          pipeline: $(resources.pipeline.images.pipelineName)
+          branchName: $(resources.pipeline.images.sourceBranch)
```

Comment:
`$(images.branchName)` is being used in two other places in this file...can be replaced with `$(resources.pipeline.images.sourceBranch)`

Reply:
Fixed.